app/services/idv/attempter.rb

```diff
@@ -9,14 +9,17 @@ def initialize(current_user)
     end
 
     def increment
-      current_user.update!(
-        idv_attempts: current_user.idv_attempts + 1,
-        idv_attempted_at: Time.zone.now
-      )
+      UpdateUser.new(
+        user: current_user,
+        attributes: { idv_attempts: current_user.idv_attempts + 1, idv_attempted_at: Time.current }
+      ).call
```

Comment:
Why the change from `Time.zone.now` to `Time.current`? I think we should be consistent and stick with one. Looks like we use the former more than the latter.